### PR TITLE
VACMS-3868 + VACMS-4281: DRUSH 10 + upgrade migrate plus and add migration tests

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #3868 & #4281 

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/phpunit/Migration/VaFormMigrationTest.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      'va_node_form',
+      'va_form',
+      file_get_contents(__DIR__ . '/fixtures/forms_updated.csv'),
+      [
+        'field_va_form_row_id' => 999999,
+        'field_va_form_title' => 'Test VA Form - Updated',
+      ],
+      1,
+      TRUE,
+    ];
```
Comment (swirtSJW) [inline tests/phpunit/Migration/VaFormMigrationTest.php:71]: Nice!  I like this test model a lot.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
@acrollet Amazing work here.   I love what you did with the tests.   That is a huge improvement.

No real changes requested but can't approve until changes are made so our deploy doesn't break due to site alert.   Also the one module version regression in .lock.

File: composer.json [hunk 2]
```diff
@@ -113,7 +113,7 @@
         "drupal/role_delegation": "^1.0@alpha",
         "drupal/seckit": "^2.0",
         "drupal/simplesamlphp_auth": "^3.2",
-        "drupal/site_alert": "1.x-dev#d3700174bdddeca1f33c6995fde86e436b034270",
+        "drupal/site_alert": "1.2",
```
[the remark this answers is not in the document]
Comment (acrollet) [reply composer.json:116]: PR for that is here: https://github.com/department-of-veterans-affairs/devops/pull/8627

File: composer.json [hunk 3]
```diff
@@ -355,9 +359,7 @@
                 "Don't attempt username sync when user was already matched by username": "https://www.drupal.org/files/issues/2020-07-31/reroll-existing-test-and-logging-fix-2748731-29.patch"
             },
             "drupal/site_alert": {
-                "Add drush commands https://www.drupal.org/project/site_alert/issues/3118800": "https://www.drupal.org/files/issues/2020-05-20/site_alert-add_drush_commands-3118800-12.patch",
-                "Make drush commands Drush 8 https://www.drupal.org/project/site_alert/issues/3118800": "https://www.drupal.org/files/issues/2020-05-20/site_alert-add_drush_commands_for_drush8-3118800-14.patch",
-                "Allow access to /ajax/site_alert route in maintenance mode": "https://www.drupal.org/files/issues/2020-04-29/maintenance-access.patch"
+                "3118800 - Add drush commands to add and remove alerts by label": "https://www.drupal.org/files/issues/2020-10-26/3118800-45.patch"
```
[the remark this answers is not in the document]
Comment (acrollet) [reply composer.json:362]: the updated command can remove by label, I tested locally.

File: composer.lock [hunk 4]
```diff
@@ -6939,26 +7176,26 @@
         },
         {
             "name": "drupal/menu_link_attributes",
-            "version": "1.2.0",
+            "version": "1.1.0",
```
[the remark this answers is not in the document]
Comment (acrollet) [reply composer.lock:7179]: nice catch, thanks!

**Review by swirtSJW — APPROVED** [review]
Approved.  Awesome work.   
I leave it up to bigger brains than mine to work out the timing of this one and https://github.com/department-of-veterans-affairs/devops/pull/8627

File: .tugboat/config.yml [hunk 5]
```diff
@@ -30,6 +30,12 @@ services:
         # cypress-axe dependencies - https://docs.cypress.io/guides/getting-started/installing-cypress.html#System-requirements
         - apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
 
+        # Install php ZIP extension.
+        - apt-get install libzip-dev zip
+        # The following line may be remove once PHP >= 7.3 is in use.
+        - docker-php-ext-configure zip --with-libzip
+        - docker-php-ext-install zip
```
Comment (ElijahLynn) [inline .tugboat/config.yml:37]: We are on PHP 7.3 now as of a few hours ago. 
```suggestion
```

File: tests.yml [hunk 6]
```diff
@@ -41,4 +41,4 @@ va/tests/check-cer:
 
 va/tests/status-error:
   description: Check for Drupal status errors
-  command: "drush $DRUSH_ALIAS core-requirements --severity=2 | grep . && exit 1 || echo 'Success: No Drupal status requirement errors were found.'"
+  command: "drush $DRUSH_ALIAS core-requirements --format=list --severity=2 | grep . && exit 1 || echo 'Success: No Drupal status requirement errors were found.'"
```
Comment (ElijahLynn) [inline tests.yml:44]: Hrm, didn't realize a list format was an option. Gonna use that in another test to check for PHP errors.

**Comment by ElijahLynn** [comment]
> time() expects exactly 0 parameters, 1 given

This should be resolved by https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4611 which I just merged in from master branch.

**Comment by ElijahLynn** [comment]
Tested this on cms-test DEV and PROD and IT WORKS!!!! LGTM!

**Review by ElijahLynn — APPROVED** [review]
Tested this on cms-test DEV and PROD and IT WORKS!!!! LGTM!

**Pull request merged**